packages/deployment/lib/kes.js

```diff
@@ -66,10 +74,24 @@ class UpdatedKes extends Kes {
    * @returns {Promise.<boolean>} returns true if successful
```

Comment:
I don't think this returns a promise, right? Just a boolean?

Reply:
It does return a promise at all times because it is a async/await function